projects/core/src/cms/model/content-slot-data.model.ts

```diff
@@ -2,5 +2,5 @@ import { ContentSlotComponentData } from './content-slot-component-data.model';
 
 export interface ContentSlotData {
   components?: ContentSlotComponentData[];
-  properties?: any;
```

Comment:
I would reconsider this change, as object here doesn't add much in terms of type safety, but make using it harder.

Reply:
Done